src/OvpJwplayer/SyncMediaHandler.php

```diff
@@ -527,6 +518,17 @@ protected function marshalParameters(Message $node, ?array $media = null): array
             }
         }
 
+        $personRefs = array_unique(array_merge(
+            $node->get('person_refs', []),
```

Comment:
@efjacobson minimize getNodes calls.  run one query to get em all, then filter them later by node type

Reply:
@gdbrown done